<?xml version="1.0"?>
<launch>

      <node name="murona_node" pkg="murona" type="murona_node" output = "screen">
         <param name="pub_motion_topic"          type="string" value="/robot_motion"     />
         <param name="pub_path_topic"            type="string" value="rona/move/path"         />
         <param name="pub_marker_topic"          type="string" value="collision_marker"  />
         <param name="pub_state_topic"           type="string" value="planner/state"     />
         <param name="pub_move_pause_topic"      type="string" value="rona/move/ctrl"        />
         <param name="sub_map_topic"             type="string" value="/map"              />
         <param name="sub_motion_topic"          type="string" value="/robot_motion"     />
         <param name="sub_target_topic"         type="string" value="target"          />
         <param name="sub_move_state_topic"      type="string" value="rona/move/state"        />
         <param name="sub_move_process_topic"    type="string" value="rona/move/process"      />
         <param name="robot_frame"               type="string" value="robot0"            />
         <param name="map_frame"                 type="string" value="map"               />
         <param name="max_share_path_lenght"     type="double" value="3.0"               />
         <param name="average_vel"               type="double" value="0.4"               />
         <param name="refresh_motion"            type="double" value="1.0"               />
         <param name="time_margin_collision"     type="double" value="5.0"               />
         <param name="robot_radius"              type="double" value="0.3"               />
         <param name="color_r"                   type="double" value="1.0"               />
         <param name="color_g"                   type="double" value="0.0"               />
         <param name="color_b"                   type="double" value="0.5"               />
         <param name="color_a"                   type="double" value="0.5"               />
         <param name="robot_id"                  type="int"    value="0"                 />
         <param name="robot_prio"                type="int"    value="0"                 />
      </node>
      
</launch>